cql3/expr/expression.cc

```diff
@@ -900,6 +1006,11 @@ expression replace_column_def(const expression& expr, const column_definition* n
                 throw std::logic_error(format("replace_column_def invalid with column tuple: {}", to_string(expr)));
             },
             [&] (const token&) { return expr; },
+            [&] (const unresolved_identifier&) { return expr; },
+            [&] (const column_mutation_attribute&) { return expr; },
+            [&] (const function_call&) { return expr; },
+            [&] (const cast&) { return expr; },
+            [&] (const field_selection&) { return expr; },
```

Comment:
In the fullness of time, we'll want to descend into nested expressions here, right?  If so, please add a TODO comment.  If not, then the function's blurb should mention that limitation.

Reply:
I'll mention it in the patch changelogs for completeness.

Reply:
Ah, with_function_call already did, I'll update the others.

Reply:
> The next-to-last patch does that.

Does what?  Does it add TODOs here or modify the blurb in expression.hh?